### PR TITLE
Just found a glitch. Changed klass to class

### PR DESCRIPTION
**Pull request by RahmatAliMalik5**
updated klass to class

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
just change klass to class in doc

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by RahmatAliMalik5** [comment]
@googlebot I signed it!

File: packages/common/src/directives/ng_style.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export class NgStyle implements DoCheck {
       private _ngEl: ElementRef, private _differs: KeyValueDiffers, private _renderer: Renderer2) {}
 
   @Input('ngStyle')
-  set ngStyle(values: {[klass: string]: any}|null) {
+  set ngStyle(values: {[class: string]: any}|null) {
```
Comment (alfaproject) [inline packages/common/src/directives/ng_style.ts:56]: `class` is a reserved word in TypeScript/JavaScript that's why we use `klass` instead.

**Comment by lazarljubenovic** [comment]
The suggested change [doesn't even compile](https://ts-ast-viewer.com/#code/MYGwhgzhAEBKD28Au0DeAoa0IFMUDsBzAZSQE8QcAKANzBAFccIAuNAbVEleyQCcAlkQC6bMPjIBfAD74GIEAEo00SejVA). Furthermore, you changed the name of the destructed symbol but not its usages. A piece of advice: it's not a good idea to do code changes blindly from the GitHub editor.

File: packages/common/src/directives/ng_style.ts (hunk 1 above)
Comment (RahmatAliMalik5) [reply packages/common/src/directives/ng_style.ts:56]: Thanks. I was thinking that was a mistake in documentation. My bad...

**Pull request closed without merging**